functions.php

```diff
@@ -27,34 +27,23 @@ function getDatabase($dbName) {
  */
 function populateTable(array $vinylDetails): string {
     $result = '';
-    foreach ($vinylDetails as $Details) {
-        if (isset($Details['artist_firstname']) &&
-            isset($Details['album']) &&
-            isset($Details['year'])) {
+    foreach ($vinylDetails as $vinyl) {
+        if (isset($vinyl['artist_firstname']) &&
+            isset($vinyl['album']) &&
+            isset($vinyl['year'])) {
 
-            if (isset($Details['cover_art'])) {
-                $coverArt = $Details['cover_art'];
-            } else {
-                $coverArt = 'no_img.jpg';
-            }
-            $artistFirstName = $Details['artist_firstname'];
-            if (isset($Details['artist_lastname'])) {
-                $artistLastName = $Details['artist_lastname'];
-            } else {
-                $artistLastName = '';
-            }
-            $album = $Details['album'];
-            $year = $Details['year'];
+            $vinyl['cover_art'] = $vinyl['cover_art'] ?? 'no_img.jpg';
+            $vinyl['artist_lastname'] = $vinyl['artist_lastname'] ?? '';
 
             $result .= ' 
                  <div class="collection_item">
                   <div>
-                     <img src="' . $coverArt . '" alt="Cover Art [If Available]">
+                     <img src="'. $vinyl['cover_art'].'" alt="Cover Art [If Available]">
```

Comment:
PSR says you should have spaces either side of the concatenation operator:
` 'string' . 'string'`

Reply:
i was worried that was what was making the image not display. but we fixed that so I will fix this.